MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -834,6 +842,22 @@ private void BuildStreamVideoItem(StreamInfo playlistItem, MediaOptions options,
 
             // Prefer matching audio codecs, could do better here
             var audioCodecs = ContainerProfile.SplitValue(audioCodec);
+
+            // Enforce HLS audio codec restrictions
+            if (string.Equals(playlistItem.SubProtocol, "hls", StringComparison.OrdinalIgnoreCase))
+            {
+                var supportedHlsAudioCodecs = new List<string> { "aac", "ac3", "eac3", "mp3" };
```

Comment:
this should be pulled out to be `static`

Create `supportedHlsAudioCodecs` and `supportedHlsAudioCodecsMp4`